core/src/lib.rs

```diff
@@ -99,12 +103,22 @@ pub type DefaultFileService = FileServiceImpl<
 
 #[derive(Debug, Serialize)]
 pub enum InitLoggerError {
+    OnTouch(String),
```

Comment:
It's unclear what this means

Reply:
`LockbookFolderCreationError`

Reply:
`lib.rs` is *the* contract for `core` described in rust, so let's be verbose here

Reply:
But realistically, this is an `UnexpectedError` there's no corrective action we want a user to have to take in this situation